common/production_methods/08_monuments.txt

```diff
@@ -6,7 +6,7 @@ pm_default_building_eiffel_tower = {
 
 	country_modifiers = {
 		workforce_scaled = {
-			country_prestige_add = 100
+			country_prestige_add = 150
```

Comment:
A bit random that you're doing monuments as part of the Ottomans, but okay :p

Reply:
Funny enough it actually mattered because changing hagia sophia's prestige count significantly determined whether or not they were a GP. I got it to make them right on the line to becoming one through these changed

Reply:
😂 Fair enough. But seems like this can easily change over the course of a campaign, so it _may_ be the wrong thing to adjust..

Reply:
I don't like these monument changes as this will further put nations without monuments behind. Also the Eiffel tower can't be build until midgame? So why are we changing it?

Reply:
I can revert them, I think the only caveat before doing so is it'll guarantee the Ottomans can't be a GP until they complete at least a few of the reforms so idk if that is good or bad for their original placement

Reply:
Why not decaying prestige modifier at the start of the game? through the journal entry.

Reply:
Yea that sounds good

Reply:
Actually all things considered, I think its appropriate to let it remain as is without the monuments so i'll proceed with just reverting.